### ISSUE TITLE
Missing argument 1 for LightOpenID::__construct()

### ISSUE THREAD

**Issue by Quix0r** [friendica/friendica#5014]
This error message comes when I use OpenId in login form:
````
( ! )  Warning: Missing argument 1 for LightOpenID::__construct(), called in  /var/www/../src/Module/Login.php on  line 93 and defined in  /var/www/../vendor/lightopenid/lightopenid/openid.php  on line 74
--


1 | 0.0000 | 363256 | {main}(  ) | .../index.php:0
2 | 0.0158 | 734864 | Friendica\Module\Login::post(  ) | .../index.php:366
3 | 0.0158 | 734272 | Friendica\Module\Login::openIdAuthentication(  ) | .../Login.php:62
4 | 0.0963 | 740000 | LightOpenID->__construct(  ) | .../Login.php:93
````

**Comment by MrPetovan** [comment]
Does it still log you in?

**Comment by Quix0r** [comment]
No, it aborts immediately.

**Comment by MrPetovan** [comment]
Closed via #5068
